### ISSUE TITLE
False positive with `useCallback`

### ISSUE THREAD

**Issue by aboqasem** [steadicat/eslint-plugin-react-memo#3]
![Screenshot 2021-10-25 at 10 26 12 PM](https://user-images.githubusercontent.com/62098043/138714574-446ec490-317f-4282-af73-728f7b196ae9.png)

**Comment by Opty1712** [comment]
<img width="1072" height="252" alt="Image" src="https://github.com/user-attachments/assets/55caa4e8-36ad-4710-bb1b-7727efce7758" />
the same

**Comment by Opty1712** [comment]
removing $ helped ($updateToolbar → updateToolbar)!

```ts
const updateToolbar = useCallback(() => {
    const selection = $getSelection();

    if ($isRangeSelection(selection)) {
      setIsBold(selection.hasFormat('bold'));
    }
  }, []);
```
